pkg/kubelet/dockershim/network/metrics/metrics.go

```diff
@@ -28,9 +28,11 @@ import (
 
 const (
 	// NetworkPluginOperationsKey is the key for operation count metrics.
-	NetworkPluginOperationsKey = "network_plugin_operations"
+	NetworkPluginOperationsKey = "network_plugin_operations_total"
```

Comment:
won't changing this cause backwards compatibility issues for existing dashboards and alerts? Or I guess looking below, this was never used before?

Are these names all following the latest guidance from the Prometheus project?

Reply:
Thanks John for the quick review. I don't see `NetworkPluginOperationsKey` being used anywhere in this file or in the kubernetes repo when I used Github-search.

The names do follow the convention from the Prometheus project - https://prometheus.io/docs/practices/naming/#metric-names. 

The reason for change from "network_plugin_operations" to "network_plugin_operations_total" was to follow this convention.